File: lógica/JS/estrutura-para/fatorial.js
// Fazer um programa para ler um número natural N (valor máximo: 15), e depois calcular e mostrar o
// fatorial de N.

var N, fat

N = parseInt(prompt('Digite o valor de N'))

fat = 1
for(var i = N; i > 0; i--) {
    fat = fat * i 
}
console.log('Fatorial = ' + fat)

File: lógica/Python/estrutura-repetitiva/fatorial.py
# Fazer um programa para ler um número natural N (valor máximo: 15), e depois calcular e mostrar o 
# fatorial de N. 

n = int(input('Digite o valor de N: '))

fatorial = 1
for i in range(n, 0, -1):
    fatorial = fatorial * i 

print(f'Fatorial = {fatorial}')